barretenberg/cpp/src/barretenberg/eccvm/eccvm_prover.cpp

```diff
@@ -303,6 +303,61 @@ template <ECCVMFlavor Flavor> void ECCVMProver_<Flavor>::execute_final_pcs_round
     PCS::compute_opening_proof(commitment_key, shplonk_output.opening_pair, shplonk_output.witness, transcript);
 }
 
+/**
+ * @brief Batch open the transcript polynomials as univariates for Translator consistency check
+ * TODO(#768): Find a better way to do this. See issue for details.
```

Comment:
If you just use these #'s it links to the wrong issue, assuming it's an issue in the monorepo. Will you put a full URL instead?